SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/TestimonialsCollectionViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  TestimonialsCollectionViewCell.swift
+//  SomosMasApp
+//
+//  Created by Fran Nasich on 01/04/2022.
+//
+
+import UIKit
+
+class TestimonialsCollectionViewCell: UICollectionViewCell {
+
+    
+    @IBOutlet weak var testimonialImage: UIImageView!
+    @IBOutlet weak var testimonialEpigraph: UILabel!
+    
+    override func awakeFromNib() {
```

Comment:
you don´t need to override a function if you do nothing different that in the parent views